docker/app.setup.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
Technically we don't have to change the name of this script. It could still be app.entrypoint.sh.

Reply:
Now that I've slept on this, I kinda like the idea of renaming it back to app.entrypoint. Makes it more obvious.

Reply:
I take that back I forgot the entrypoint.sh file still exists and is needed cause it invokes the `exec $@`.